src/core/shape_inference/include/sparse_fill_empty_rows_shape_inference.hpp

```diff
@@ -0,0 +1,125 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <unordered_set>
+
+#include "openvino/op/sparse_fill_empty_rows.hpp"
+#include "utils.hpp"
+
+namespace ov {
+namespace op {
+namespace v16 {
```

Comment:
```suggestion
namespace ov::op::v16 {
```
Maybe there are other places in added code when this can be used?